ee/codegen/src/__test__/nodes/__snapshots__/inline-subworkflow-node.test.ts.snap

```diff
@@ -20,7 +20,11 @@ class InlineSubworkflowNodeDisplay(BaseInlineSubworkflowNodeDisplay[InlineSubwor
     target_handle_id = UUID("3fe4b4a6-5ed2-4307-ac1c-02389337c4f2")
     workflow_input_ids_by_name = {}
     node_input_ids_by_name = {}
-    output_display = {}
+    output_display = {
+        InlineSubworkflowNode.Outputs.final_output: NodeOutputDisplay(
+            id=UUID("edd5cfd5-6ad8-437d-8775-4b9aeb62a5fb"), name="final-output"
+        )
+    }
```

Comment:
This was the main goal of the PR:
- `id` matches that of the output variable
- `name` matches the original name in the UI
- `output_display[str]` matches the attribute name generated